geomstats/learning/mdm.py

```diff
@@ -0,0 +1,99 @@
+"""The MDM classifier on manifolds."""
+
+import geomstats.backend as gs
+from geomstats.learning.frechet_mean import FrechetMean
+
+
+class RiemannianMinimumDistanceToMeanClassifier():
+    """
+    Classifier implementing the MDM scheme on manifolds.
+
+    Parameters
+    ----------
+    riemannian_metric : string or callable.
+        The distance metric to use.
+
+    Attributes
+    ----------
+    G: array-like, shape=[n_classes, n_features] if point_type='vector'
+                   shape=[n_classes, n_features, n_features]
+                    if point_type='matrix'
+       Frechet means of each class.
+    """
+
+    def __init__(
+            self,
+            riemannian_metric,
+            point_type='matrix'):
+        self.riemannian_metric = riemannian_metric
+        self.point_type = point_type
+        self.G = None
+
+    def fit(self, X, Y):
+        """
+        Compute Frechet mean of each class.
+
+        :param X: array-like, shape=[n_samples, n_features]
+                              if point_type='vector'
+                              shape=[n_samples, n_features, n_features]
+                              if point_type='matrix'
+                  Training data, where n_samples is the number of samples
+                  and n_features is the number of features.
+        :param Y: array-like, shape=[n_samples, n_classes]
+                  Training labels, where n_classes is the number of classes.
+        """
+        n_classes = Y.shape[-1]
+        mean_estimator = FrechetMean(
+            metric=self.riemannian_metric,
+            point_type=self.point_type)
+        frechet_means = []
+        for c in range(n_classes):
+            data_class = self.split_data_in_classes(X, Y, c)
+            # frechet_means.append(mean_estimator.fit(data_class).estimate_[0])
```

Comment:
Left-over comment?

Reply:
The class `FrechetMean` was being modified during my modifications, so I kept the old way, I'll delete it on final push =).

Reply:
Ok!

Reply:
@DanielBrooks75 a friendly reminder